src/ocean/task/util/StreamProcessor.d

```diff
@@ -184,6 +184,7 @@ class StreamProcessor ( TaskT : Task )
 
         if (throttler_config.resume_point == size_t.max)
             throttler_config.resume_point = total / 5;
+		else
```

Comment:
Wouldn't it better instead to remove the `else` from the previous if-block (i.e. always execute both the `enforce`s)? If you go this way, the additional scope surrounding both the enforces can also be removed.